openslides_backend/action/actions/user/user_mixin.py

```diff
@@ -238,3 +238,157 @@ def check_meeting_and_users(
         for meeting_id, users in meeting_users.items():
             users.append(user_fqid)
             assert_belongs_to_meeting(self.datastore, users, int(meeting_id))
+
+
+class UpdateHistoryMixin(Action):
+    def get_history_information(self) -> Optional[List[str]]:
+        informations: List[str] = []
+
+        info_update = False
+        all_meetings: List[Set[str]] = []
+        all_groups_added: List[Set[int]] = []
+        all_groups_removed: List[Set[int]] = []
```

Comment:
Use a set directly here instead of lists of sets. This way, you can easily check below if only one item was added/removed

Reply:
I don't want to mix groups of different meetings.

Reply:
Okay, I use a set for the three temporary results.